### ISSUE TITLE
/proc/self/mounts has entry even after usb unmount

### ISSUE THREAD

**Issue by krishnaclouds** [rbrito/usbmount#11]
Even after removing the usb, /proc/self/mounts is not being updated. It still holds the entries to the old mount points. After 8 successive usb insertions, new pen drive fails to mount.

**Comment by mathieulj** [comment]
Could you supply more information (ex. which distribution are you using, which version, what shows up in the logs) please? Otherwise it's pretty much a shot in the dark trying to figure out whats going wrong. Everything works on my Debian Stretch machines no mater how many times I cycle the connection. Logs that may help with debugging the issue are `sudo dmesg -w`, `sudo journalctl -fu usbmount*` and `sudo tailf /var/log/messages`.

**Comment by krishnaclouds** [comment]
> #  This file is part of systemd.
> #
> #  systemd is free software; you can redistribute it and/or modify it
> #  under the terms of the GNU Lesser General Public License as published by
> #  the Free Software Foundation; either version 2.1 of the License, or
> #  (at your option) any later version.
> #
> # Entries in this file show the compile time defaults.
> # You can change settings by editing this file.
> # Defaults can be restored by simply deleting this file.
> #
> # See systemd-system.conf(5) for details.
> 
> [Manager]
> #LogLevel=info
> #LogTarget=journal-or-kmsg
> #LogColor=yes
> #LogLocation=no
> #DumpCore=yes
> #ShowStatus=yes
> #CrashChangeVT=no
> #CrashShell=no
> #CrashReboot=no
> #CtrlAltDelBurstAction=reboot-force
> #CPUAffinity=1 2
> #JoinControllers=cpu,cpuacct net_cls,net_prio
> #RuntimeWatchdogSec=0
> #ShutdownWatchdogSec=10min
> #CapabilityBoundingSet=
> #SystemCallArchitectures=
> #TimerSlackNSec=
> #DefaultTimerAccuracySec=1min
> #DefaultStandardOutput=journal
> #DefaultStandardError=inherit
> #DefaultTimeoutStartSec=90s
> #DefaultTimeoutStopSec=90s
> #DefaultRestartSec=100ms
> #DefaultStartLimitIntervalSec=10s
> #DefaultStartLimitBurst=5
> #DefaultEnvironment=
> #DefaultCPUAccounting=no
> #DefaultIOAccounting=no
> #DefaultBlockIOAccounting=no
> #DefaultMemoryAccounting=no
> #DefaultTasksAccounting=yes
> #DefaultTasksMax=
> #DefaultLimitCPU=
> #DefaultLimitFSIZE=
> #DefaultLimitDATA=
> #DefaultLimitSTACK=
> #DefaultLimitCORE=
> #DefaultLimitRSS=
> #DefaultLimitNOFILE=
> #DefaultLimitAS=
> #DefaultLimitNPROC=
> #DefaultLimitMEMLOCK=
> #DefaultLimitLOCKS=
> #DefaultLimitSIGPENDING=
> #DefaultLimitMSGQUEUE=
> #DefaultLimitNICE=
> #DefaultLimitRTPRIO=
> #DefaultLimitRTTIME=
> cat: '/etc/systemd/system.conf.d/*.conf': No such file or directory
> cat: /run/systemd/system.conf: No such file or directory
> cat: '/run/systemd/system.conf.d/*.conf': No such file or directory
> cat: /lib/systemd/system.conf: No such file or directory
> cat: '/lib/systemd/system.conf.d/*.conf': No such file or directory

**Comment by krishnaclouds** [comment]
@mathieulj Any Progress with the logs!!

**Comment by mathieulj** [comment]
I'm sorry but for the moment I'm a little stumped. I have not found a way to reproduce this issue and nothing in your configuration explains what we saw. The command works manually and the disconnect actually runs the command (we know so by the logs) yet it does not seem to be working when run by the disconnect. My only suspicion at the moment is that the log contained other entries between insertion and remove that may be pertinent. From the formatting it seems some entries were removed (I assume to keep them more brief and concise).
